### DISCUSSION TITLE
Scrolling

### DISCUSSION THREAD

**Discussion by CkreaM** [TodePond/CellPond#298] [Ideas]
So, I've been using cellpond for fun and noticed I can't scroll, I have to zoom out which is really annoying, can you please add this?
I know what you're going to say "CTRL + SCROLL" but, that just zooms out, I want proper scrolling, plus if you zoom out too much you can zoom back in.

**Comment by loglot** [comment]
"ALT + SCROLL"

**Reply by CkreaM** [reply]
That just zooms out the cellpond game, I wanna scroll down the cellpond RULES, again, CTRL + SCROLL just zooms out

**Reply by CkreaM** [reply]
I just found out it's because of the browser I'm using.. To the owner: Can you make it so this also works with Opera GX?

**Reply by loglot** [reply]
> That just zooms out the cellpond game, I wanna scroll down the cellpond RULES, again, CTRL + SCROLL just zooms out

ALT, not CTRL

**Reply by CkreaM** [reply]
Yeah opera opens the opera gx menu when I press alt, so it cancels anything that happens in a browser game, doesn't matter anymore

**Comment by CkreaM** [comment]
Actually forget it
